historianutils/historianutils.go

```diff
@@ -105,6 +108,14 @@ func MaxInt64(a int64, b int64) int64 {
 	return b
 }
 
+// RoundFloat returns the closest integer to the given float number.
+func RoundFloat(val float64) int32 {
```

Comment:
Why not use https://golang.org/pkg/math/#Round
math.Round()

Reply:
I am not using that function because its return type is still float64, while in my case I want to use int32. I will rename this function to RoundFloatToInt and make use of the math.Round() function inside.